test/spec/tokenManager.js

```diff
@@ -52,6 +57,191 @@ define(function(require) {
       });
     });
 
+    describe('refresh', function() {
+      it('allows refreshing an idToken', function(done) {
+        return oauthUtil.setupFrame({
+          oktaAuthArgs: {
+            url: 'https://auth-js-test.okta.com',
+            clientId: 'NPSfOkH5eZrTy8PMDlvx',
+            redirectUri: 'https://auth-js-test.okta.com/redirect'
+          },
+          tokenManagerAddKeys: {
+            'test-idToken': {
+              idToken: 'testInitialToken',
+              claims: {'fake': 'claims'},
+              expiresAt: 1449699930,
```

Comment:
Is it a problem to hardcode the expiration here? Or is the idea that this is already in the past, so we have to run the refresh always?